README.adoc

```diff
@@ -3,137 +3,359 @@
 
 == Introduction
 
-This repository contains everything needed to release https://github.com/jenkinsci/jenkins[Jenkins core]:
+This repository contains everything needed to release https://github.com/jenkinsci/jenkins[Jenkins core] or some Jenkins components libraries.
+
+This includes:
 
 * Jenkinsfiles for Releasing and Packaging.
-* Kubernetes POD templates definition for releasing and packaging on Linux and Windows environments.
+* Kubernetes POD templates definition for releasing and packaging environments on Linux and Windows operating systems.
 * Any configuration or utility scripts
 
 The release process implies following Git repository:
 
 * https://github.com/jenkinsci/jenkins[jenkinsci/jenkins] or any other git repository containing the Jenkins code to be released.
 * https://github.com/jenkinsci/packaging[jenkinsci/packaging] contains tools and configuration to package Jenkins for OS distribution [Debian, Red Hat, SUSE, MSI, WAR]
-* https://github.com/jenkins-infra/chart[jenkins-infra/chart] contains Helm chart used to deploy release.ci.jenkins.io on top of Kubernetes.
+* https://github.com/jenkins-infra/chart[jenkins-infra/chart] contains Helm chart used to deploy link:release.ci.jenkins.io[release.ci.jenkins.io] on top of Kubernetes.
 * https://github.com/jenkins-infra/azure[jenkins-infra/azure] contains the terraform code used to provision infrastructure needed by the release environment
 
-== Release
+== Settings
```

Comment:
not sure if this is one of the most important categories, should we move it down to the bottom as an appendix ?

Reply:
I was wondering the same so maybe it makes more sense to move it down